### PR TITLE
LSTM

### PR DESCRIPTION
**Pull request by youdit**
Created a LSTM model for dataset (In Tensorflow 2.0 keras API), Accuracy=67%

### CODE REVIEW

**Comment by youdit** [comment]
i have added 2 layer LSTM to reach 75% accuracy
